packages/langium/src/lsp/semantic-token-provider.ts

```diff
@@ -196,28 +198,28 @@ export abstract class AbstractSemanticTokenProvider implements SemanticTokenProv
         this.clientCapabilities = clientCapabilities;
     }
 
-    semanticHighlight(document: LangiumDocument, _params: SemanticTokensParams, cancelToken = CancellationToken.None): SemanticTokens {
+    async semanticHighlight(document: LangiumDocument, _params: SemanticTokensParams, cancelToken = CancellationToken.None): Promise<SemanticTokens> {
```

Comment:
These changes are breaking. Are they "mild" enough to be included in 1.1 nonetheless?

Reply:
There is very little reason for developers to override these methods. I think it's mild enough for a 1.1.